src/geobr.jl

```diff
@@ -0,0 +1,640 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+Provides functions to download data from the GeoBR database:
+
+`GeoBR.state`
+`GeoBR.municipality`
+`GeoBR.region`
+`GeoBR.country`
+`GeoBR.amazon`
+`GeoBR.biomes`
+`GeoBR.disasterriskarea`
+`GeoBR.healthfacilities`
+`GeoBR.indigenousland`
+`GeoBR.metroarea`
+`GeoBR.neighborhood`
+`GeoBR.urbanarea`
+`GeoBR.weightingarea`
+`GeoBR.mesoregion`
+`GeoBR.microregion`
+`GeoBR.intermediateregion`
+`GeoBR.immediateregion`
+`GeoBR.municipalseat`
+`GeoBR.censustract`
+`GeoBR.statisticalgrid`
+`GeoBR.conservationunits`
+`GeoBR.semiarid`
+`GeoBR.schools`
+`GeoBR.comparableareas`
+`GeoBR.urbanconcentrations`
+`GeoBR.poparrangements`
+`GeoBR.healthregion`
+"""
+module GeoBR
+
+using GeoIO
+using DataDeps
+using CSV
+using Tables
+using TableTransforms
+
+const APIVERSIONS = (v"1.7.0",)
+
+"""
+    metadata()
+
+Retrieve the metadata file path for the GeoBR project.
+
+# Returns
+The file path of the metadata CSV file.
+"""
+function metadata()
+    url = "http://www.ipea.gov.br/geobr/metadata/metadata_1.7.0_gpkg.csv"
+    ID = "GeoBR_metadata"
+    try
+        @datadep_str filepath(ID, url)
```

Comment:
Are you sure the logic is correct here? you are registering a dataset with ID, and then querying the dataset from disk with `filepath(ID, url)`, this will always cause repeated downloads?

Can you please follow the NaturalEarth template as much as possible? This metadata function should return the `ID` as the path, and the `get` function can then append the filename inside the folder.

Reply:
It's actually working as intended. The second @datadep_str call won't re-download if the first one succeeded. It's just a fallback in case the initial try fails.